Riot/AppDelegate.m

```diff
@@ -4246,8 +4266,15 @@ - (void)addCallStatusBar:(NSString*)buttonTitle
     
     [_callStatusBarButton setTitle:buttonTitle forState:UIControlStateNormal];
     [_callStatusBarButton setTitle:buttonTitle forState:UIControlStateHighlighted];
+    if (@available(iOS 11.0, *))
+    {
+        UIEdgeInsets safeAreaInsets = UIApplication.sharedApplication.keyWindow.safeAreaInsets;
+        safeAreaInsets.top = topBarSize.height - CALL_STATUS_BAR_HEIGHT;
+        safeAreaInsets.bottom = 0.0;
+        _callStatusBarButton.titleEdgeInsets = safeAreaInsets;
+    }
```

Comment:
We can move this behavior to `statusBarDidChangeFrame` method.

Reply:
👍